.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
You cannot make this change due to https://github.com/ciena-blueplanet/ember-test-utils/issues/86